app/controllers/pets_controller.rb

```diff
@@ -64,14 +59,16 @@ def print
     if @pet.image.present?
       render layout: 'print'
     else
-      redirect_to pet_path(@pet), notice: 'No picture to print, Please upload a Picture.'
+      redirect_to pet_path(@pet), notice: 'No picture to print,
```

Comment:
Don't put the string on multiple lines. Put the arguments of `redirect_to` on multiple lines.

```ruby
redirect_to(
  pet_path(@pet),
  notice: '...'
)
```